runtime/vm/j9vm.tdf

```diff
@@ -994,3 +994,12 @@ TraceEvent=Trc_VM_simulateStack_RTVSEND_JBinvokestaticsplit_index Overhead=1 Lev
 TraceEvent=Trc_VM_simulateStack_RTVSEND_JBinvokespecialsplit_index Overhead=1 Level=3 Template="simulateStack RTV_SEND JBinvokespecialsplit index(%zu)"
 TraceEvent=Trc_VM_simulateStack_RTVSEND_JBinvokedynamic_classSig Overhead=1 Level=5 Template="simulateStack RTV_SEND JBinvokedynamic classSig(%.*s)"
 TraceEvent=Trc_VM_simulateStack_RTVSEND_others_classSig Overhead=1 Level=5 Template="simulateStack RTV_SEND others classSig(%.*s)"
+
+TraceEntry=Trc_VM_snapshot_readSnapshotFromFile_Entry NoEnv Overhead=1 Level=1 Template="readSnapshotFromFile() Snapshot Heap = %p. File being read = %s"
+TraceExit=Trc_VM_snapshot_readSnapshotFromFile_Exit NoEnv Overhead=1 Level=1 Template="readSnapshotFromFile() Snapshot read successful"
+TraceEntry=Trc_VM_snapshot_writeSnapshotToFile_Entry NoEnv Overhead=1 Level=1 Template="writeSnapshotToFile() Snapshot Heap = %p. File being written to = %s"
+TraceExit=Trc_VM_snapshot_writeSnapshotToFile_Exit NoEnv Overhead=1 Level=1 Template="writeSnapshotTofile() Snapshot write successful"
+TraceEntry=Trc_VM_snapshot_subAllocateSnapshotMemory_Entry NoEnv Overhead=1 Level=1 Template="subAllocateMemory() Snapshot Header = %p. Byte amount = %d."
+TraceExit=Trc_VM_snapshot_subAllocateSnapshotMemory_Exit NoEnv Overhead=1 Level=1 Template="subAllocateMemory() Memory allocated = %p."
+TraceEvent=Trc_VM_snapshot_warmClassLoadFromSnapshot_ClassLoadHookFailed Overhead=1 Level=1 Template="loadWarmClassFromSnapshot() Warm class load hook failed class=%p, %s"
+TraceEvent=Trc_VM_snapshot_loadWarmClassFromSnapshot_ClassInfo Overhead=1 Level=1 Template="loadWarmClassFromSnapshot() LoadClass clazz=%p, %s"
```

Comment:
Not all of these new tracepoints are actually used; please remove the unused ones.